packages/core/src/render3/util/stringify_utils.ts

```diff
@@ -8,6 +8,7 @@
 
 import {Type} from '../../interface/type';
 import {getComponentDef} from '../def_getters';
+import {ClassDebugInfo} from '../interfaces/definition';
```

Comment:
```suggestion
import type {ClassDebugInfo} from '../interfaces/definition';
```
Should fix the circular import issue.